packages/paste-core/primitives/box/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "peerDependencies": {
     "@twilio-paste/animation-library": "^0.3.2",
+    "@twilio-paste/box": "^7.0.0",
```

Comment:
Why does the box package depend on itself? lol Did you mean to add this to the hbs instead? https://github.com/twilio-labs/paste/pull/2737/files#diff-a68998e6714d0c1caeee57cc875a41564aff96679f3bee3a10b70aa1c596fc83R29

Reply:
ugh yes 🙃 thanks for catching this!